cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/concepts/http/HttpRequest.kt

```diff
@@ -34,4 +34,6 @@ class HttpRequest(
     val arguments: List<Node>,
     val httpMethod: HttpMethod,
     override val concept: HttpClient,
-) : HttpClientOperation(underlyingNode = underlyingNode, concept = concept)
+) : HttpClientOperation(underlyingNode = underlyingNode, concept = concept) {
+    val calls = mutableListOf<HttpEndpoint>()
```

Comment:
In the ontology we called this `to`. Maybe better suitable?

Reply:
renamed